webpack/assets/javascripts/react_app/components/notifications/drawer/notificationGroup.js

```diff
@@ -39,9 +40,17 @@ export default (
               onClickedLink={onClickedLink}
               key={notification.id}
               notification={notification}
-              onMarkAsRead={onMarkAsRead.bind(this, group)}
+              onMarkAsRead={onMarkAsRead.bind(this, group, [notification.id])}
```

Comment:
if we make a seperate action for marking group, this change is will not be required.